models/basic_cnn.py

```diff
@@ -26,7 +26,7 @@ def __init__(self, name="basic_cnn", subspectrogram_points = config.SUBSPECTROGR
 
         self.dense1 = Dense(200,activation=tf.nn.relu,name=f"{name}_dense1")
 
-        self.dense2 = Dense(number_of_emotions,activation=tf.nn.sigmoid,name=f"{name}_output")
```

Comment:
You might want to keep the sigmoïd activation if the loss is mse

Reply:
Added back on